### ISSUE TITLE
Titles on posts created with New the Photo Post shortcut are started as lowercase but should start as uppercase

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12905]
Steps to reproduce:

1. Using iPhone 6S or later, find the WordPress app icon and "force tough" it to see shortcuts.
1. Select "New Photo Post".
1. Choose an image (or a few).
1. Press on "Insert" at the bottom right corner.
1. Add a title.

**Result:** I expected the title to start as uppercase, however, if I use the New Photo Post shortcut with the block editor then the title is started as lowercase. The same problem does not happen with the classic editor. ([2m15s](https://cloudup.com/cT5VXkLiPgI))

<img src="https://user-images.githubusercontent.com/1119271/68436929-a429d980-017c-11ea-9940-4a0023e2eb4d.png" width="270" alt="new-photo-post-add-title">
<sup>Tested with WP Internal 13.6.20111104 (HockeyApp version) on iPhone 6S iOS 13.2.</sup>

**Comment by designsimply** [comment]
@etoledom this came up in 13.5 beta testing but that cycle has ended and 13.6 is now in code freeze. Would it be possible for you to fix this for 13.7?

**Comment by etoledom** [comment]
If I'm not mistaken, @koke had a fix for this one?

**Comment by koke** [comment]
Yes, there is a fix in https://github.com/wordpress-mobile/gutenberg-mobile/issues/1530 that should be ready today

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
